code-challenges.test.js

```diff
@@ -49,12 +173,93 @@ const studyMinutesWeek2 = {
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
+
+
 // b) Create the function that makes the test pass.
+// Pseudo Code:
+  //The parameter will be an object array
+  // Create a function called, sanchezToAli,that takes in an object and returns an array of the values sorted from least to greatest utilizing .sort((a, b) => a - b). 
+  // Inputs: objects, studyMinutesWeek1 & studyMinutesWeek2.
+  // Outputs: an array of given object values sorted from low to highest value.  Expected const studyMinutesWeek1Output = [15, 15, 20, 30, 30, 60, 90] & const studyMinutesWeek2Output = [10, 15, 20, 45, 60, 65, 100].
+  // Finally run jest to verify function.
+
+    //*** Below are just console.logs that utilize the given objects and still call the correct answer without writing a function.  I just did this by accident trying to stack them to see what it would tell me.  
+    // console.log((Object.values(studyMinutesWeek1)).sort((a, b) => a - b))
+    // console.log((Object.values(studyMinutesWeek2)).sort((a, b) => a - b))
+
+
+// This does work in Jest with the given paramenters and outputs the expected values as stated in the prompts.  Will not work on Jest since Jest is stating it is not a function.  Both of the below and above work on thier own just not with Jest due to the object array issue where returned info is not just an array but an object and .sort should only work on arrays.  It still works however Jest is not buying it as a function.  
+
+// My sanchezToAli function below, or non-function.
+    // const sanchezToAli = (array) => array.sort((a, b) => a - b);
+
+    function sanchezToAli (array) {
+        return array.sort((a, b) => a - b);
+      }
+
+  console.log(sanchezToAli(Object.values(studyMinutesWeek1)))
+  console.log(sanchezToAli(Object.values(studyMinutesWeek2)))
```

Comment:
Your function should do the work that your console.log and your tests are doing in restructuring the object.